fairscale/nn/data_parallel/gossip/graph_manager.py

```diff
@@ -0,0 +1,290 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
```

Comment:
@vtantia @msbaines we'll need to change the license I think

Reply:
We generally use the wording:

# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
#
# This source code is licensed under the BSD license found in the
# LICENSE file in the root directory of this source tree.